client/layout/global-sidebar/menu-items/notifications/index.jsx

```diff
@@ -135,8 +143,31 @@ class SidebarNotifications extends Component {
 			'is-initial-load': this.state.animationState === -1,
 		} );
 
+		const shouldShowNotificationsPointer =
+			// Show pointer for 2 weeks.
+			Date.now() < Date.parse( '23 May 2024' ) &&
+			// Show pointer to users registered before 08-May-2024 (when we moved the notifications to the footer).
+			this.props.currentUserId < 250450000 &&
+			// Show pointer only if translated.
+			( englishLocales.includes( this.props.locale ) ||
+				hasTranslation( 'Looking for your notifications? They have been moved here.' ) );
```

Comment:
Why is it better to hide the message than showing it un-translated? Should we block merging this PR until we have the text translated?

Reply:
> Why is it better to hide the message than showing it un-translated?

Because untranslated content is a pain point for our customers (pxLjZ-5Ir-p2, p58i-8e8-p2)

> Should we block merging this PR until we have the text translated?

I think it's better if English users see this change immediately